queen-attack/example.nim

```diff
@@ -1,18 +1,12 @@
-type
-  Position* = tuple[rank, file: int]
-
-proc isValid(white, black: Position): bool =
+proc isValid(white, black: tuple[rank, file: int]): bool =
```

Comment:
@dom96 I changed it because I was worried that the implementer might not see that they would need to implement the Position type. Now all they have to do is just provide (int, int) tuples without knowing that they are Positions. But you seem to know Nim better than I; maybe it's not a big deal.